source/openapi-admin-v3.yaml

```
@@ -605,6 +605,88 @@ paths:
       - "$ref": "#/components/parameters/GroupId"
       - "$ref": "#/components/parameters/AppId"
       - "$ref": "#/components/parameters/ServiceId"
+  "/groups/{groupId}/apps/{appId}/services/{datasourceId}/commands/validate_documents":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/DatasourceId"
+    post:
+      tags:
+        - schemas
+      operationId: adminValidateDocuments
+      summary: Sample & Validate Documents
+      description: |-
+        Query a subset of documents in a collection and validate each match against a schema.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              required:
+                - database_name
+                - collection_name
+                - from_schema
+              properties:
+                database_name:
+                  type: string
+                  description: The name of the database that contains the collection to validate.
+                collection_name:
+                  type: string
+                  description: The name of the collection to validate.
+                from_schema:
+                  type: object
+                  description: The EJSON schema to validate sampled documents against.
+                query:
+                  type: object
+                  default: {}
+                  description: A MongoDB query for the collection. Use this to include a specific subset of documents in the sample.
```

```suggestion
                  description: A MongoDB query for the collection. Use this to include a specific subset of documents in the collection for the sample.
```

and could you provide a link for the text "MongoDB Query"?